graphics/graphics-text.md

```diff
@@ -0,0 +1,39 @@
+# Displaying text
+
+The Graphic Entity Module includes two different class to display text, here's how they work.
+
+## Text
+
+The basic class to display text, it will be displayed as a label on you screen.
+
+```java
+graphicEntityModule.createText("Hello World")
+    .setFontFamily("Lato")
+    .setStrokeThickness(5) // Adding an outline
+    .setStrokeColor(0xffffff) // a white outline
+    .setFontSize(50)
+    .setFillColor(0x000000); // An setting the text color to black
+```
+You'll see in this example that you have a lot of options to manipulate the text, but this class has one downside :
+the font is not garanteed to be the good one, since the client browser has to provide it.
+So it doesn't support a lot of fonts and your font will probably be replaced by the default one of the browser.
+
+## BitmapText
+
+This class is used to display text using a bitmap font in your asset folder.
+But if you have no font (or doesn't set one) your text will not be displayed.
```

Comment:
I feel this line is unnecessary (also you meant "don't")

Reply:
guess you're right i'm over explaining a bit